casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -556,37 +547,19 @@ object BlockAPI {
     )
   }
 
-  private def getBlockInfo[A, F[_]: Monad: BlockStore](
+  private def getBlockInfo[A, F[_]: Applicative: BlockStore](
       block: BlockMessage,
-      constructor: (
-          BlockMessage,
-          Float
-      ) => A
-  )(implicit casper: MultiParentCasper[F]): F[A] =
-    for {
-      dag <- casper.blockDag
-      // TODO this is temporary solution to not calculate fault tolerance all the blocks
-      oldBlock = dag.latestBlockNumber.map(_ - block.body.state.blockNumber).map(_ > 100)
-      // Old block fault tolerance / invalid block has -1.0 fault tolerance
-      normalizedFaultTolerance <- oldBlock.ifM(
-                                   dag
-                                     .isFinalized(block.blockHash)
-                                     .map(isFinalized => if (isFinalized) 1f else -1f),
-                                   Float.MinValue.pure[F]
-                                 )
-      initialFault   <- casper.normalizedInitialFault(ProtoUtil.weightMap(block))
-      faultTolerance = normalizedFaultTolerance - initialFault
-
-      blockInfo = constructor(block, faultTolerance)
-    } yield blockInfo
+      constructor: (BlockMessage, Float) => A
+  ): F[A] = constructor(block, -1f).pure[F]
```

Comment:
Might be its better to put here `Float.minValue` to emphasise this is a plug and not some normal value expected from deprecated SafetyOracle.

Reply:
This is to satisfy what's expected in tests. But we will remove fault tolerance completely so I don't consider it's important.